docs/pages/reference/config.mdx

```diff
@@ -768,6 +768,31 @@ kubernetes_service:
       command: ['curl', 'http://metadata.google.internal/computeMetadata/v1/instance/attributes/cluster-name', '-H', 'Metadata-Flavor: Google']
       period: 1m0s
 
+## This section configures the 'discovery service'
+discovery_service:
+    enabled: yes
+    aws:
+       # AWS resource types. Valid options are:
+       # 'ec2 - discovers and registers AWS EC2 instances
+     - types: ["ec2"]
+       # AWS regions to register EC2 instances from
+       regions: ["eu-central-1"]
+       # AWS resource tags to match when registering EC2 instances
+       tags:
+         "*": "*"
+       # install is used to provide parameters to the AWS SSM Document
+       install:
```

Comment:
Are the `install:` and `join_params:` sections required? If not, can we notate them with "Optional"?